.github/workflows/portal-tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Portal Unit Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Change directory to portal
+        run: cd portal
+
+      - uses: pnpm/action-setup@v4
+        name: Install pnpm
+        with:
+          version: 8
+          run_install: false
```

Comment:
That is because I wanted to separate those steps of `pnpm setup` and `pnpm install. 
I can change it to `run_install: true`, it won't make a significant difference.

Reply:
Alright I included the `run_install: true` to be more concise.

Reply:
Nice, thanks! was just wondering what the difference was, but this seems to work well.